src/app/adminLayout/admin-home/admin-home.component.html

```diff
@@ -1 +1 @@
-Home works :)
+<h1>Admin Home works :)</h1>
```

Comment:
Tu rozumiem jest placeholder dla jakichś newsów etc. lub cokolwiek innego co powinien widzieć admina po zalogowaniu?

Reply:
tak, to jest placeholder dla admin-hode, kiedyś ustalimy jak on będzie miał wyglądać to to napiszemy